wireapp/wire-ios-mono
# feat: Implement Folder Creation UI & Wire-iOS Integration - WPB-12012

This PR implements the SwiftUI-based folder picker UI and its integration with Wire-iOS, building upon the previously added domain types. 

## Changes

### UI Package (WireMoveToFolderUI)

- Add SwiftUI views for folder picker:
  - `CreateFolder`: Main view with navigation and empty state handling

## UI/UX
- Maintains existing create folder functionality
- Uses Wire's semantic colors and styling
- Proper error handling for folder operations

## To Test
1. Open conversation context menu
2. Select "Move to Folder"
3. Tap plus button
4. Create new folder
5. Tap Create
6. Verify folder has been created and moved to that folder
---

### Checklist


---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
